frontend/src/components/ProjectConfiguration.vue

```diff
@@ -101,14 +101,15 @@
         </b-form-checkbox>
       </b-form-group>
       <b-form-group label="Auto elevate to annotator"
-        description="Automatically allows annotation on real dataset after the training and testing stage. A disabled stage counts as having been completed.">
+                    description="Automatically allows annotation on real dataset after the training and testing stage. A disabled stage counts as having been completed.">
```

Comment:
?

Reply:
The description attribute has just been aligned with the label attribute. Or is there something wrong with the description?

Reply:
Nope nothing wrong, sorry I was just wondering why the indent, but makes sense that it's for alignment.